packages/order-experience-api/src/openapi.yaml

```diff
@@ -499,6 +502,9 @@ paths:
                       type: string
                     namespace:
                       type: string
+                    lastValidPurchaseDateTime:
+                      type: string
+                      format: date-time
```

Comment:
Could you add the description to here also?

Reply:
This is a good idea, also mention of the expected time format.